src/pyfaf/rpm.py

```diff
@@ -80,7 +79,8 @@ def store_rpm_deps(db, package, nogpgcheck=False):
         new.flags = p.Flags()
         evr = p.EVR()
         if evr:
-            new.epoch, new.version, new.release = rpmutils.stringToVersion(evr)
+            def evr(self):
```

Comment:
you defined this function, but you don't seem to use it. Also you removed setting of `new.epoch, new.version, new.release`.

Reply:
There is also names collison